torch_glow/src/PyTorchModelLoader.h

```diff
@@ -41,8 +79,12 @@ class PyTorchModelLoader {
   /// Mapping from PyTorch Values to Glow NodeValues created during loading.
   using ValueMap =
       std::unordered_map<const torch::jit::Value *, glow::NodeValue>;
+  // TODO: delete this once it's replace by the following 2 maps.
   ValueMap valueMap_;
 
+  std::unordered_map<const torch::jit::Value *, std::unique_ptr<ValueMapping>>
```

Comment:
If ValueMapping class is movable, not need to have a std::unique_ptr wrapper.

Reply:
I did this because ValueMapping hands out pointers to it's internals so those need to remain valid. I will move the unique_ptr to inside ValueMapping